CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # safe-nd - Change Log
 
+## [0.6.0]
+
+- Change CI to GitHub Actions.
+- Clean up explicitly-listed non-warn lints.
+- Add `Keypair` and remove BLS methods for full IDs.
+- Remove `SecretKey`.
```

Comment:
SecretKey was never available. It was added and removed in a separate commit just to preserve history.

Reply:
Updated, thanks! :)